module/VuFind/tests/integration-tests/src/VuFindTest/Mink/HoldsTest.php

```diff
@@ -646,15 +646,18 @@ public function testEditingDifferentPickupLocations()
         // Place the second hold:
         $page = $this->gotoRecordById('dollar$ign/slashcombo');
         $this->placeHoldAndGoToHoldsScreen($page, ['#pickUpLocation' => 'C']);
+        //        sleep(5);
```

Comment:
I think these commented lines are not necessary.

Reply:
Should have seen this as well. Sorry. Fixed